R/skip.R

```diff
@@ -239,9 +229,37 @@ skip_on_bioc <- function() {
 #'   [`R-base.pot`](https://github.com/wch/r-source/blob/master/src/library/base/po/R-base.pot).
 #' @rdname skip
 skip_if_translated <- function(msgid = "'%s' not found") {
-  if (gettext(msgid, domain = "R") == msgid) {
-    return(invisible(TRUE))
+  if (gettext(msgid, domain = "R") != msgid) {
+    skip(paste0("\"", msgid, "\" is translated"))
+  } else {
+    invisible()
   }
+}
 
-  skip(paste0("\"", msgid, "\" is translated"))
+#' Superseded skip functions
+#'
```

Comment:
Add the lifecycle badge?

Reply:
Maybe mention in NEWS?

Reply:
They were already superseded; I'm just pulling out into a separate file.